src/main/resources/jupyter/edit-mode.js

```diff
@@ -0,0 +1,396 @@
+const dialog = require('base/js/dialog')
+const utils = require("base/js/utils")
+
+define(() => {
+    // TEMPLATED CODE
+    // Leonardo has logic to find/replace templated values in the format $(...).
+    const googleProject = $(googleProject);
```

Comment:
I don't think Leo will subtitute quotes for you, should probably be `"$(googleProject)"`